content/tutorials/themes/theme-objects/breadcrumb/index.md

```diff
@@ -11,11 +11,11 @@ links:
 
 # Breadcrumb Theme Object Introduction
 
-The Breadcrumb Theme Object displays the path to the currently selected page in the menu structure like this:
-RootPage > SubPage > SubSubPage
-The starting level, separator and styliong are configurable.
+The Breadcrumb Theme Object displays the path to the currently selected page in the menu structure like this:/n
+RootPage > SubPage > SubSubPage/n
+The starting level, separator and styling are configurable.<br/>
```

Comment:
```suggestion
The starting level, separator and styling are configurable.
```
HTML `<br/>` not necessary for a new line.

Reply:
Well, I added those because \n seems to be ignored. If you compare the current source with what the site shows, it seems to ignore \n The only visible new lines are in the table below which has <br> in them
![DnnDocsNewLine](https://user-images.githubusercontent.com/8999382/95127009-9cd9ca00-0757-11eb-8930-26fc51bb4e3e.jpg)

Reply:
`<br/>` is HTML.  These are DocFx flavored markdown files.  The `>` may be causing issues, but the simple fix is to just make sure you have two carriage returns between these three lines.  For this particular line, the carriage return was working just fine.  It was the previous ones that were not.  ;-)

Reply:
Hmm, I'm not sure I fully understand. 
I'm missing something here I guess.
Do we need two line breaks for one to show on the site?
So this markdown:

```
The Breadcrumb Theme Object displays the path to the currently selected page in the menu structure like this:

RootPage > SubPage > SubSubPage

The starting level, separator and styling are configurable.
```


should show up as


```
The Breadcrumb Theme Object displays the path to the currently selected page in the menu structure like this:
RootPage > SubPage > SubSubPage
The starting level, separator and styling are configurable.
```

on dnndocs?

The reason I ask and why I started adding <br> is because ATM I think

```
The Breadcrumb Theme Object displays the path to the currently selected page in the menu structure like this:
RootPage > SubPage > SubSubPage
The starting level, separator and styling are configurable.
```

renders without breaks on dnndocs